disco_aws_automation/disco_cloudfront.py

```diff
@@ -0,0 +1,194 @@
+import botocore
+import boto3
+import random
+from ConfigParser import ConfigParser
+import uuid
+
+from . import normalize_path
+
+DEFAULT_CONFIG_FILE_CLOUDFRONT = "disco_cloudfront.ini"
+
+class DiscoCloudfront(object):
+    """ Class to create Cloudfront distribution  """
+
+    def __init__(self, vpc, config_file=None):
```

Comment:
Doesn't look like the `config_file` parameter is being respected.  How about defaulting it to `DEFAULT_CONFIG_FILE_CLOUDFRONT` here and always setting `self.config_file = config_file` below?